src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -58,6 +58,9 @@ public void processRequest(List<String> args) throws Exception {
       List<File> javaJarFiles = filterFilesByExtension(jarFiles, ".java");
 
       String[] scalaSources = GenericWorker.appendToString(ops.files, scalaJarFiles);
+      // include java files from .srcjar's to support mixed compilation
+      scalaSources = GenericWorker.appendToString(scalaSources, javaJarFiles);
```

Comment:
I'm cool with that. I was thinking about introducing a new variable but hesitated to reduce the potential of error for future errors if someone references `scalaSources` when they meant to reference `appendSrcJarSources`.